main.star

```diff
@@ -11,7 +11,7 @@ cdk_erigon_package = "./cdk_erigon.star"
 databases_package = "./databases.star"
 deploy_zkevm_contracts_package = "./deploy_zkevm_contracts.star"
 ethereum_package = "./ethereum.star"
-optimism_package = "github.com/ethpandaops/optimism-package/main.star@1.2.0"
```

Comment:
Looks like they push new commits to the image but don't release new tags.. Too bad, we can use latest commit from `main` for now :)